z3c/dependencychecker/main.py

```diff
@@ -38,6 +43,13 @@ def parse_command_line():
         default=False,
         help='Show debug output',
     )
+    parser.add_option(
+        '--exit-zero',
+        action='store_false',
+        dest='exit_status',
+        default=True,
```

Comment:
Having a positive variable with a default of True always helps a lot when reasoning about it. So: good!

(Stuff like `if omit_statuscode == False` always makes my head hurt.)